packages/latino/build.sh

```diff
@@ -2,10 +2,10 @@ TERMUX_PKG_HOMEPAGE=https://www.lenguajelatino.org/
 TERMUX_PKG_DESCRIPTION="Lenguaje de programación de código abierto para latinos y de habla hispana."
```

Comment:
Why not keep description in English so that package search mechanism work better.

Reply:
Since this is a language aimed at Spanish speakers, I think the description is fine.